task/prefetch-dependencies/0.1/prefetch-dependencies.yaml

```
@@ -30,7 +34,27 @@ spec:
     type: string
     description: The name of the key in the ConfigMap that contains the CA bundle data.
     default: ca-bundle.crt
+
+  stepTemplate:
+    env:
+    - name: CONFIG_FILE
+      value: $(params.config-file)
+    volumeMounts:
+      - name: config
+        mountPath: /mnt/config
+
   steps:
+  - name: sanitize cachi2 config file with yq
```

```suggestion
  - name: sanitize-cachi2-config-file-with-yq
```
I think step names must be valid kubernetes names